Engine/DataFeeds/SubscriptionDataReader.cs

```diff
@@ -208,6 +209,16 @@ public void Initialize()
                 }
             }
 
+            // If Fred data, set the access token in data factory
+            var fred = _dataFactory as FredApi;
```

Comment:
One day we'll add an interface for this and the world will breathe a sigh of relief from setting all the auth codes generically. 🦄 🌈 🌞